data/processed/industry/employment_culture_entertainment_recreation.yaml

```diff
@@ -0,0 +1,30 @@
+api_doc_url: https://www.nomisweb.co.uk/api/v01/help
+api_type: POST
+data_date: 20200218
+description_short: Employment in cultural, entertainment and leisure industries
+description: This indicator measures level of employment in cultural, entertainment and leisure industries based on the Business Register Employment Survey. Those sectors are identified as clusters of SIC-4 (industry) codes
+endpoint_url: http://www.nomisweb.co.uk/api/v01/dataset/
+framework_group: place_potential
+is_experimental: True
+order: [year, nuts_id, nuts_year_spec, value.id] # USE AS IS: it specifies the order of the exported indicator fields
```

Comment:
minor, remove the comment (just because it doesn't apply anymore in the schema template, sorry)